src/org/anjocaido/groupmanager/permissions/AnjoPermissionsHandler.java

```diff
@@ -181,19 +175,13 @@ public String getGroup(String userName) {
 		}
 		
 		// Process overridden permissions
-		
-		Iterator<String> itr = overrides.iterator();
-		
-		while (itr.hasNext()) {
-			
-			String node = itr.next();
 
-			if (playerPermArray.contains("-" + node)) {
-				playerPermArray.remove("-" + node);
-			}
-			
+		for (String node : overrides) {
+
+			playerPermArray.remove("-" + node);
+
 			playerPermArray.add(node);
-			
+

```

Comment:
Why change to a for loop over an Iterator?

Reply:
Requires less lines of code? If it's a performance issue or something, please notify

Reply:
No reason other than curious.